bin/codacy-coverage.js

```diff
@@ -41,30 +41,17 @@
             return;
         }
 
-        var token = program.token || process.env.CODACY_REPO_TOKEN,
-            commitId = program.commit,
-            format = program.format || 'lcov',
-            pathPrefix = program.prefix || '';
-
-        if (!token) {
-            return loggerImpl.error(new Error('Token is required'));
-        }
-
-        // Parse the coverage data for the given format and retrieve the commit id if we don't have it.
-        return Q.all([lib.getParser(format).parse(pathPrefix, input), getGitData.getCommitId(commitId)]).spread(function (parsedCoverage, commitId) {
-            // Now that we've parse the coverage data to the correct format, send it to Codacy.
-            loggerImpl.trace(parsedCoverage);
-            lib.reporter({
-                endpoint: program.endpoint
-            }).sendCoverage(token, commitId, parsedCoverage).then(function () {
-                loggerImpl.debug('Successfully sent coverage');
-            }, function (err) {
-                loggerImpl.error('Error sending coverage');
-                loggerImpl.error(err);
-            });
-        }, function (err) {
-            loggerImpl.error(err);
-        });
+        var opts = {
+            endpoint: program.endpoint,
+            token: program.token,
+            commitId: program.commit,
+            format: program.format,
+            pathPrefix: program.prefix,
+            verbose: program.verbose,
+            debug: program.debug
+        };
+
+        return lib.handleInput(input, opts);
```

Comment:
Since this is going to be used from the CLI we just need to output some notification that the process has completed like we did before. All it should need to do is: 

```js
return lib.handleInput(input, opts).then(function() {
  loggerImpl.debug('Successfully sent coverage');
}, function(err) {
  loggerImpl.error('Error sending coverage');
  loggerImpl.error(err);
});
```

This way there's some notification when the process completes in the logs.